torchao/float8/float8_utils.py

```diff
@@ -53,7 +52,7 @@ def amax_to_scale(
     # to care about this for float32/bfloat16.
     if orig_dtype is torch.float16:
         res = torch.clamp(res, max=torch.finfo(torch.float16).max)
-    scale.copy_(res)
+    return res.float()
```

Comment:
nit: .to(torch.float32)

Reply:
sure. For my own context, why is that better?

Reply:
IDK but but `.float()` reminds me of `.cuda()` or `FloatTensor()` which all kinda feel outdated? Completely optional